### PR TITLE
Explain relationship with existing QPP API

### PR DESCRIPTION
**Pull request by marimiyachi**
https://jira.cms.gov/browse/QPPA-365

Very succinctly explained, but can be more verbose if you want @lucasmbrown-usds. My plan next is to link from the developer docs just explaining what qpp-measures-data is, but not mention the relationship with the existing QPP API in that text. LMK what you think.

Reviewer: @kencheeto (I spellchecked myself this time :P)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,19 @@
 # qpp-measures-data
-Quality Payments Program Measures Data Repository
+Quality Payment Program Measures Data Repository
```
Comment (kencheeto) [inline README.md]: maybe we can use this line as the header?

Comment (marimiyachi) [inline README.md]: sgtm!

Comment (lucasmbrown-usds) [inline README.md]: also @marimiyachi @kencheeto i don't think we ever implemented this comment?

Comment (marimiyachi) [inline README.md]: I didn't push the commit but if you look at the README the change was made :)

**Pull request closed without merging**